events/activities/hold_court_activity/hold_court_events_general.txt

```diff
@@ -37265,37 +37235,9 @@ scripted_trigger hold_court_8200_infidel_trigger = {
 	is_available_ai_adult = yes
 	NOT = { faith = root.faith }
 	NOR = {
-
 		# Warcraft
-		AND = {
-			religion = { is_in_family = rf_light }
-			root.faith = { has_doctrine = tenet_light_syncretism }
-		}
-		AND = {
-			religion = { is_in_family = rf_shadow }
-			root.faith = { has_doctrine = tenet_shadow_syncretism }
-		}
-		AND = {
-			religion = { is_in_family = rf_disorder }
-			root.faith = { has_doctrine = tenet_disorder_syncretism }
-		}
-		AND = {
-			religion = { is_in_family = rf_order }
-			root.faith = { has_doctrine = tenet_order_syncretism }
-		}
-		AND = {
-			religion = { is_in_family = rf_life }
-			root.faith = { has_doctrine = tenet_life_syncretism }
-		}
-		AND = {
-			religion = { is_in_family = rf_death }
-			root.faith = { has_doctrine = tenet_death_syncretism }
-		}
-		AND = {
-			religion = { is_in_family = rf_elemental }
-			root.faith = { has_doctrine = tenet_elemental_syncretism }
-		}
-
+		save_temporary_scope_as = infidel
+		root.faith = { NOT = { is_syncretic_with_faith = { FAITH = scope:infidel.faith } } }
 	}
```

Comment:
NOT inside NOR thus it's twice no. Should be just without NO.
Pay attention to logic and test after the changes.